src/Microsoft.DotNet.Tools.Publish/PublishCommand.cs

```diff
@@ -14,124 +14,74 @@
 
 namespace Microsoft.DotNet.Tools.Publish
 {
-    public class Program
+    public class PublishCommand
     {
-        public static int Main(string[] args)
-        {
-            DebugHelper.HandleDebugSwitch(ref args);
+        public string ProjectPath;
+        public string Configuration;
+        public string OutputPath;
+        public string Framework;
+        public string Runtime;
+        public NuGetFramework NugetFramework;
+        public IEnumerable<ProjectContext> ProjectRidFxPermutations;
```

Comment:
For public APIs we expect people to consume our guideline is to use properties and not fields. See https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/framework-design-guidelines-digest.md  "DO NOT expose public fields. Use properties instead."